surveys/templates/partials/multiselect_geometry_widget.html

```diff
@@ -118,6 +118,7 @@
           e.preventDefault();
           jsonLayer.eachLayer(function(layer) {
             layer.setStyle(layerStyle);
+            layer.feature.properties.selected = false;
```

Comment:
The lack of this line was causing styles to go a little haywire after the user hit "Clear all". Because we weren't setting `selected = false` on the actual `layer` object, the layer would be deselected from the form, but it would refuse to be styled correctly by `resetHightlight()` on user mouseout.